pkgs/development/libraries/webkitgtk/2.4.nix

```diff
@@ -44,6 +44,10 @@ stdenv.mkDerivation rec {
   '';
   patches = [
     ./webcore-svg-libxml-cflags.patch
+    (fetchurl {
```

Comment:
please also use `fetchpatch` here

Reply:
Done. I hope @globin or you will explain when to use `fetchpatch`: https://github.com/NixOS/nixpkgs/pull/28634#discussion_r135500791

Reply:
Always for patches. In some case like yours it might be fine. But that way it is more consistent.